tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
what i meant was, conceptually what is the purpose of that production in the JSX spec, if the recommended transpiler doesn't support it?